Riot/Modules/TabBar/TabBarCoordinator.swift

```diff
@@ -18,14 +18,24 @@
 
 import UIKit
 
+/// TabBarCoordinator input parameters
+class TabBarCoordinatorParameters {
```

Comment:
Question: can we use a struct here as it is used only from swift?

Reply:
Good question atm we only used reference type for coordinator parameters as parameters properties are mainly reference type. But it is possible to use struct (value type) with reference type properties (classes)